pkg/agent/controller/egress/egress_controller.go

```diff
@@ -466,6 +551,29 @@ func (c *EgressController) syncEgress(egressName string) error {
 		return err
 	}
 
+	if egress.Spec.ExternalIPPool != "" && c.cluster.LocalNodeJoined() {
+		if c.cluster.ShouldSelectEgress(egress) {
+			if !c.localIPDetector.IsLocalIP(egress.Spec.EgressIP) {
+				a := NewNodeEgressIPAssigner(c.cluster.NodeConf)
+				if err := a.AssignOwnerNodeEgressIP(egress.Spec.EgressIP); err != nil {
+					return err
+				}
+			}
+			if err := c.updateEgressStatus(egress, c.nodeName); err != nil {
+				return fmt.Errorf("update egress status error:%v", err)
+			}
+			klog.V(2).InfoS("Assigned owner node for egress and update status",
```

Comment:
This should only be logged when the node was first assigned the IP? otherwise it will be logged everytime the egress is processed which is confusing

Reply:
I think we can change log level here or delete the log, as AssignOwnerNodeEgressIP and updateEgressStatus both logged the detail.